index.html

```diff
@@ -25,70 +25,310 @@
+                     <dl class="list container">
+                      <dt class="list-item">
+                          <a href="https://twitter.com/haywayalive" target="_blank" rel="noopener noreferrer"><i class="fa fa-twitter"></i></a>
+                      </dt>
+                      <dd></dd>
+                      <dt class="list-item">
+                          <a href="https://www.linkedin.com/in/abubakar-ummar-4b6643245" target="_blank" rel="noopener noreferrer"><i class="fa fa-linkedin"></i></a>
+                      </dt>
+                      <dd></dd>
+                      <dt class="list-item">
+                          <a href="https://medium.com/@haywayaheadshotstrategy.fx" target="_blank" rel="noopener noreferrer"><i class="fa fa-medium"></i></a>
+                      </dt>
+                      <dd></dd>
+                      <dt class="list-item">
+                          <a href="https://github.com/Haywayaheadshot" target="_blank" rel="noopener noreferrer"><i class="fa fa-github"></i></a>
+                      </dt>
+                      <dd></dd>
+                      <dt class="list-item">
+                          <a href="https://angel.co/u/abubakar-ummar" target="_blank" rel="noopener noreferrer"><i class="fa fa-angellist"></i></a>
+                      </dt>
+                      <dd></dd>
+                      
+                  </dl>
+                  
+                  </section>
+            </main>
+      </div>
+      <div class="portfolio-container page-one">
+        <header class="pageone-header">
+          <img src="images/SnapshootPortfolio.svg" alt="A snapshoot Portfolio Header From Figma">
+        </header>
+        <section class="pageone-body">
+            <h1><strong>Tonic</strong></h1>
+            <section>
+              <h4 class="subhead">CANOPY</h4>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">Front End Dev</h6>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">2022</h6>
+            </section>
+            <section class="pageone-text">
+              <div>daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+            </section>
+            <section class="buttons-section">
+              <ul >
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">html</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">css</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">javascript</button>
+                </li>
+              </ul>
+            </section>
+            <section>
+              <button type="button" class="footer-button">See Project</button>
```

Comment:
- Kindly make sure to design this see project button and look exactly the same as how it looks to the provided Figma. Try to change your `padding` and `border-radius` and adjust it to design our button properly. 😉 

   - ### Your design
  ![y-button](https://user-images.githubusercontent.com/87197412/185700177-ef480c96-541f-4816-aa1c-1a5707d08ca7.PNG)

   - ### Figma design
  ![r-button](https://user-images.githubusercontent.com/87197412/185700190-b2a81591-2c1d-45c8-85ed-c46177688448.PNG)